### DISCUSSION TITLE
[Bug Report] Making another cpu hang on whilst getting on-top of a platform causes the cpu to still hang on.

### DISCUSSION THREAD

**Discussion by RunninPigeon** [TrianglyRU/OrbinautFramework#34] [General]
![Orbinaut Framework 2 9_9_2024 7_08_47 PM](https://github.com/user-attachments/assets/8c59b3be-b681-41e5-a474-84477c2830d4)

**Comment by TrianglyRU** [comment]
Will be fixed in 2024.10
